example/benchmark.html

```diff
@@ -52,7 +52,7 @@
     // If the size of each message is small, send/receive multiple messages
     // until the sum of sizes reaches this threshold.
     minTotal: getIntFromInput('mintotal'),
-    multipliers: getIntArrayFromInput('multipliers'),
+    multipliers: getFloatArrayFromInput('multipliers'),
```

Comment:
Could you change |multipliers| in example/fetch_benchmark.html and example/xhr_benchmark.html?